### PR TITLE
Add documentation on building using path

### PR DESCRIPTION
**Pull request by TechnologyClassroom**
I added a link to install rust and a command to install from source using path.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,13 +152,21 @@ brew install pastel
...
+### Via cargo (source)
+
+If you do not have cargo, install using [rust's installation documentation](https://doc.rust-lang.org/book/ch01-01-installation.html).
 
 If you have Rust 1.35 or higher, you can install `pastel` from source via `cargo`:
 ```
 cargo install pastel
 ```
 
+If you have Rust 1.35 or higher, you can install `pastel` from source via `cargo` using this repository:
```
Comment (sharkdp) [inline README.md]: I don't think we have to repeat "if you have Rust 1.35 or higher" here. Also, `cargo install` installs "from source" as well, so I would suggest to rephrase this to:

*Alternatively, you can install `pastel` from this repository using*

What do you think?

Comment (TechnologyClassroom) [reply README.md]: Sure.  Sounds good.

I deleted the source repository so I cannot make the change.
